awscli/examples/ivs-realtime/create-storage-configuration.rst

```diff
@@ -1,21 +1,21 @@
-**To create a composition storage configuration**
-
-The following ``create-storage-configuration`` example creates a composition storage configuration with the specified properties. ::
-
-    aws ivs-realtime create-storage-configuration \
-        --name "test-sc" --s3 "bucketName=test-bucket-name"
-
-Output::
-
-    {
-        "storageConfiguration": {
-            "arn": "arn:aws:ivs:ap-northeast-1:123456789012:storage-configuration/ABabCDcdEFef",
-            "name": "test-sc",
-            "s3": {
-                "bucketName": "test-bucket-name"
-            },
-            "tags": {}
-        }
-    }
-
+**To create a composition storage configuration**
+
+The following ``create-storage-configuration`` example creates a composition storage configuration with the specified properties. ::
+
+    aws ivs-realtime create-storage-configuration \
+        --name "test-sc" --s3 "bucketName=amzn-s3-demo-bucket-name"
```

Comment:
```suggestion
        --name "test-sc" --s3 "bucketName=amzn-s3-demo-bucket"
```
As discussed offline, the trailing ``-name`` needs to be removed on these updates.